snare.py

```diff
@@ -251,6 +256,19 @@ def handle_request(self, request, payload):
             response.add_header('Content-Type', content_type)
         if content:
             response.add_header('Content-Length', str(len(content)))
+
+        # logging of ip, stat, req, user, len, time
+        req = request.method + ' /' + self.run_args.page_dir + request.path
+        major = response.version.major
+        minor = response.version.minor
+        ver = " HTTP/" + str(major) + '.' + str(minor)
+        req = req + ver
+        user = getpass.getuser()
+        tim = strftime("%d/%b/%Y:%H:%M:%S %z")
+        d = {'hostIP': request.headers['Host'], 'stat': response.status, 'req': req, 'user': user,
+             'content_length': response.headers['Content-Length'], 'time': tim}
+        self.logger.info(' ', extra=d)
```

Comment:
I think this part with logging should executed only if we have `logging==True`. In other case this line will be executed with default self.logger of aiohttp server

Reply:
@afeena pls review.